iz_importer.py

```diff
@@ -144,7 +147,7 @@ def _get_exif_mapping(self, attachment_properties_map):
         }
 
         # Remove keys with None values
-        return {k: v for k, v in exif_mapping.items() if v is not None}
+        return {k: v for k, v in exif_mapping.items()}
```

Comment:
this this line still work as intended as removing none values

Reply:
Actually, we want to allow None values - there's an EXIF field that we're needing to blank out.